bigchaindb/consensus.py

```diff
@@ -139,12 +139,21 @@ def validate_transaction(bigchain, transaction):
                 raise exceptions.OperationError(
                     'Only federation nodes can use the operation `CREATE`')
 
+            # validate digital asset
+            # A create transaction will have only one output
+            asset = transaction['transaction']['asset']
+            assets.validate_asset_creation(asset['data'], asset['divisible'], asset['updatable'],
+                                           asset['refillable'], transaction['transaction']['conditions'][0]['amount'])
+            # TODO: Validate uuid?
```

Comment:
> I think that this would be solved once we improve the asset creation where a client would contact several nodes instead on only one.

In that case though, assuming we keep the backlog algorithm the same and assign each transaction to only one node in the end, wouldn't we still be vulnerable to that one node manipulating the asset? (In fact, that one node could do anything with the transaction it receives; I'm sure you and @ttmc have talked about the fault tolerance of this... it's probably in an issue / project somewhere? For now, are we assuming we can trust each deployed node?)